cmake/Modules/FindwxWidgetsTB.cmake

```diff
@@ -497,13 +497,13 @@ if(wxWidgets_FIND_STYLE STREQUAL "win32")
     if(MINGW)
       set(WX_LIB_DIR_PREFIX gcc)
     else()
-	  if(CMAKE_GENERATOR STREQUAL "Visual Studio 10 2010")
+	  if(CMAKE_VS_PLATFORM_TOOLSET STREQUAL "v100")

```

Comment:
Why not just take the value (or first four characters to deal with the `_xp` suffix) of `CMAKE_VS_PLATFORM_TOOLSET` and remove the if / then / else cascade altogether?